### PR TITLE
Add wabt compatibility test

### PR DESCRIPTION
**Pull request by duongnhn**
As mentioned in https://github.com/wasdk/wasmparser/issues/33 and https://bugs.chromium.org/p/chromium/issues/detail?id=1066386, we want to generate WAT that is wabt-compatible.

Some existing ones are not wabt-compatible so we exclude those.

### CODE REVIEW

**Comment by duongnhn** [comment]
@bmeurer @ktran PTAL :)

File: wabt-compatibility.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+// Run wabt over .out files.
+readdirSync(TEST_FOLDER)
+  .filter(fileName => extname(fileName) === ".out" && INCOMPATIBLE_FILES.indexOf(fileName) === -1)
+  .forEach(fileName => {
+    test(`wabt ${fileName}`, () => {
+      const filePath = join(TEST_FOLDER, fileName);
+      let data = new Uint8Array(readFileSync(filePath));
+      const { buffer } = parseWat(fileName, data).toBinary({
+        write_debug_names: true
```
Comment (bmeurer) [inline wabt-compatibility.spec.ts:47]: This doesn't seem to be necessary for the test.

Comment (duongnhn) [reply wabt-compatibility.spec.ts]: fixed, thanks :)

**Comment by yurydelendik** [comment]
Thank you for the patch!

**Pull request merged**